src/desktop/apps/auction/components/layout/auction_info/Registration.js

```diff
@@ -21,6 +21,10 @@ function Registration(props) {
     return null
   }
 
+  const registerText = userNeedsIdentityVerification
+    ? "Identity verification required to bid"
```

Comment:
We also want to add the FAQ link (and open it in a new tab per the Jira requirements): https://artsyproduct.atlassian.net/browse/AUCT-940